src/applications/vaos/components/AddToCalendar.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import {
+  getAppointmentTypeHeader,
+  getAppointmentDate,
+  generateICS,
+} from '../utils/appointment';
+
+export default class AddToCalendar extends React.Component {
+  render() {
+    const { appointment, facility } = this.props;
```

Comment:
I hesitate to say this given that you made 1k lines of unit test changes in another file, but we need some tests for this component. We can make a separate ticket for that.

Reply:
Ok.

Reply:
Ticket created. #4890